Models/Task.cs

```diff
@@ -0,0 +1,40 @@
+﻿
+using System.ComponentModel.DataAnnotations;
+
+namespace Fiks.Models
+{
+    public class Task
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [Required]
+        [StringLength(80)]
```

Comment:
80 je trochu málo, co to zvednout? Zrovna u tohohle, kdy tam asi bude odstavec až dva popisu